# debian_alsaconf_improve_intro.patch by Thomas Hood
#
# Improve introductory message
Index: alsa-utils-1.0.18/alsaconf/alsaconf.in
===================================================================
--- alsa-utils-1.0.18.orig/alsaconf/alsaconf.in	2008-11-02 20:59:26.000000000 +0100
+++ alsa-utils-1.0.18/alsaconf/alsaconf.in	2008-11-02 20:59:26.000000000 +0100
@@ -458,9 +458,7 @@
     Advanced Linux Sound Architecture (ALSA) driver.
 
 
-  If ALSA is already running, you should close all sound
-  apps now and stop the sound driver.
-  alsaconf will try to do this, but it's not 100%% sure." $version)
+  You should stop all sound applications now." $version)
   $DIALOG --msgbox "$msg" 20 63 || acex 0
 }
 
